includes/admin/class-wc-payments-admin.php

```diff
@@ -863,4 +882,36 @@ public function show_woopay_payment_method_name_admin( $order_id ) {
 
 		<?php
 	}
+
+	/**
+	 * Adds a notification badge to the Payments > Disputes admin menu item to
+	 * indicate the number of disputes that need a response.
+	 */
+	public function add_disputes_notification_badge() {
+		global $submenu;
+
+		if ( ! isset( $submenu[ self::PAYMENTS_SUBMENU_SLUG ] ) ) {
+			return;
+		}
+
+		$disputes_summary = $this->database_cache->get_or_add(
+			Database_Cache::DISPUTES_SUMMARY_KEY,
+			[ $this->payments_api_client, 'get_disputes_summary' ],
+			// We'll consider all array values to be valid as the cache is only invalidated when it is deleted or it expires.
+			'is_array'
+		);
+
+		$disputes_needing_response = isset( $disputes_summary['statuses']['needs_response'] ) ?? 0;
```

Comment:
looks like this line is one of the culprit. `isset()` returns `true` here. The null coalescing operator handles the `isset` for you already so can be replaced with
```suggestion
		$disputes_needing_response = $disputes_summary['statuses']['needs_response'] ?? 0;
```
However this will only show the `needs_response` and not the `warning_needs_response` status which likely need to be shown as well.

Reply:
> looks like this line is one of the culprit. isset() returns true here. 

thanks 🙈 I'm still new to using this operator obviously 😆 

> However this will only show the needs_response and not the warning_needs_response status which likely need to be shown as well.

yeah good point. I haven't been able to find how a dispute transitions into the warning_needs_response status and the docs are pretty light on details as far as I could see.

Reply:
I was using test cards from [stripe](https://stripe.com/docs/testing#disputes) 
Looks like `4000000000005423` will trigger the `warning_needs_response` status

Reply:
I've pushed [4a6943a](https://github.com/Automattic/woocommerce-payments/pull/4326/commits/4a6943a95b5fc485ac9fa6cfb285104c1c6014dd) to address this. It now sums all disputes with the status 'needs_response' and 'warning_needs_response.